### ISSUE TITLE
LoadError: UndefVarError: blasfunc not defined

### ISSUE THREAD

**Issue by alsam** [ApproxFun/ApproxFun.jl#292]
Happy New Year!

I'm getting the following error message for `Version 0.5.0-dev+1949 (2016-01-01 18:25 UTC)` of `julia` with both `v0.1.0` and TOT versions of `ApproxFun` package, when precompiling it:
``` julia
julia> using ApproxFun
INFO: Precompiling module ApproxFun...
ERROR: LoadError: LoadError: LoadError: UndefVarError: blasfunc not defined
 [inlined code] from /home/user/.julia/v0.5/ApproxFun/src/LinearAlgebra/hesseneigs.jl:6
 in anonymous at no file:0
 in include at ./boot.jl:262
 in include_from_node1 at ./loading.jl:401
 in include at ./boot.jl:262
 in include_from_node1 at ./loading.jl:401
 in include at ./boot.jl:262
 in include_from_node1 at ./loading.jl:401
 [inlined code] from boot.jl:265
 in anonymous at no file:0
 in eval at ./boot.jl:265
 [inlined code] from ./sysimg.jl:14
 in process_options at ./client.jl:222
 in _start at ./client.jl:301
while loading /home/user/.julia/v0.5/ApproxFun/src/LinearAlgebra/hesseneigs.jl, in expression starting on line 5
while loading /home/user/.julia/v0.5/ApproxFun/src/LinearAlgebra/LinearAlgebra.jl, in expression starting on line 15
while loading /home/user/.julia/v0.5/ApproxFun/src/ApproxFun.jl, in expression starting on line 20
ERROR: Failed to precompile ApproxFun to /home/user/.julia/lib/v0.5/ApproxFun.ji
 in error at ./error.jl:21
 in compilecache at loading.jl:480
 in require at ./loading.jl:339
 in eval at ./boot.jl:265
```
Thanks!

**Comment by MikaelSlevinsky** [comment]
Looks like the latest commit to Julia's base/linalg/blas.jl changed `blasfunc` to `@blasfunc`. https://github.com/JuliaLang/julia/commit/15c70cb5fe3395653941a28819006104f963eee1

**Comment by dlfivefifty** [comment]
Right now 0.5 is not supported..maintaining 0.4pre support was tiring and so I took a break.

But maybe it's time to add the 0.5 tests to Travis and try to maintain compatibility?  Hopefully there won't be as many deprecations this time

**Comment by alsam** [comment]
Thanks for the hint and for the clarification @MikaelSlevinsky and @dlfivefifty.
I tried to change `blasfunc` to `@blasfunc` and it works for me, at least precompiles the package.

**Comment by MikaelSlevinsky** [comment]
I guess @dlfivefifty fixed this on development with 6c257fb82dcf2b14417b6ad2a2306ca2d4bd9c7f, d8cdb94d624bc2a0e661de32d53961167f7f4477, 7f4fd520f1419843c3331ffb3f0b228ff1803ad4, 76de44643ac7700cd0cea048c809674d94251dd5, bfa8e7dad8ca40d8e4befed3af127be63283520a ?

**Comment by dlfivefifty** [comment]
Yep!  0.5 is still not completely working though.
